src/components/structured-list/structured-list.js

```diff
@@ -47,13 +49,9 @@ class StructuredList extends mixin(createComponent, initComponentBySearch, handl
     return array.indexOf(arrayItem) + direction; // returns -1, 0, 1, 2, 3, 4...
   }
 
-  _getInput(index) {
-    const rows = [...this.element.querySelectorAll(this.options.selectorRow)];
-    return this.element.ownerDocument.querySelector(this.options.selectorListInput(rows[index].getAttribute('for')));
-  }
-
   _handleInputChecked(index) {
-    const input = this._getInput(index);
+    const rows = [...this.element.querySelectorAll(this.options.selectorRow)];
```

Comment:
Spread operator here won't be needed as ondex reference of `NodeList` should work.

Reply:
thanks for pointing that out, I just moved it over from the `_getInput` method